spec/draft_spec.rb

```diff
@@ -63,10 +63,9 @@
       FileUtils.touch path
     end
 
-    it "raises an error" do
-      expect(lambda {
-        capture_stdout { described_class.process(args) }
-      }).to raise_error("A draft already exists at _drafts/an-existing-draft.md")
+    it "displays a warning" do
+      output = capture_stdout { described_class.process(args) }
```

Comment:
Does `abort_with` not kill the test process?

Reply:
I guess it does, the tests are passing but are wrong. If I change the string in the test, it still passes. How should I test that we abort?

Reply:
I think we can simply `warn` and `return` instead of a hard `abort`..

Reply:
With a simple `return` file gets overwritten. I thought this was workin' after testing it locally but this isn't, shouldn't have merged 😕